index.js

```diff
@@ -176,25 +176,29 @@ function redisStore(args) {
    * @param {Function} cb - A callback that returns a potential error and the response
    */
   self.get = function(key, options, cb) {
-    if (typeof options === 'function') {
-      cb = options;
-      options = {};
-    }
-    options = options || {};
-    options.parse = true;
+    return new Promise(function(resolve, reject) {
```

Comment:
Done, setting `Promise` at the top of `redisStore` so all existing uses of `Promise` will also use the users implementation if it exists. https://github.com/dial-once/node-cache-manager-redis/pull/69/commits/a362cb54d88bbc92685c564abf448f0b8db253ee